dwave/system/composites/virtual_graph.py

```diff
@@ -127,6 +137,13 @@ def __init__(self, sampler, embedding,
                  flux_bias_max_age=3600):
 
         super(VirtualGraphComposite, self).__init__(sampler, embedding)
+        warn(
+                "'VirtualGraphComposite' is deprecated due to improved calibration "
+                "of newer QPUs and in future will raise an exception; if needed, "
+                "follow the instructions in the shimming tutorial at "
+                "https://github.com/dwavesystems/shimming-tutorial instead. ",
+                DeprecationWarning
```

Comment:
I think we might need to set the `stacklevel` for the warning to be shown from the correct code line?

Reply:
I saw that we are inconsistent. [LeapHybridDQMSampler](https://github.com/dwavesystems/dwave-system/blob/b233941673cda08c194d769b067ab06ce43ea077/dwave/system/samplers/leap_hybrid_sampler.py#L480) sets the `stacklevel` but [LazyEmbeddingComposite](https://github.com/dwavesystems/dwave-system/blob/b233941673cda08c194d769b067ab06ce43ea077/dwave/system/composites/embedding.py#L579) does not. I think we need that to override Python's newer default that doesn't show warnings unless users set them to show. Do we want to do that?